jbmc/regression/jbmc-strings/ConstantEvaluationContains/Main.java

```diff
@@ -0,0 +1,15 @@
+public class Main {
+    public void contains() {
+        String str = "abcde";
+        assert str.contains("cd");
+    }
+
+    public void notContains() {
+        String str = "abcde";
+        assert !str.contains("cd");
```

Comment:
This doesn't really test anything more than `contains`, except that `operator!` is working -- instead I think you should test some cases where `contains` should return false. `assert !str.contains("fgh")`, `assert !str.contains("abcdef")`, and so on -- things that should evaluate to `false` for different reasons.